docs/metrics.md

```diff
@@ -166,39 +166,49 @@ soroban.host-fn-op.read-entry                | meter     | number of entries rea
 soroban.host-fn-op.write-entry               | meter     | number of entries written
 soroban.host-fn-op.read-key-byte             | meter     | number of key bytes in read entries
 soroban.host-fn-op.write-key-byte            | meter     | number of key bytes in written entries
-soroban.host-fn-op.read-ledger-byte          | meter     | number of data + code bytes in read entries
+soroban.host-fn-op.read-ledger-byte          | meter     | number of entry (data + code) bytes in read entries
 soroban.host-fn-op.read-data-byte            | meter     | number of data bytes in read entries
 soroban.host-fn-op.read-code-byte            | meter     | number of code bytes in read entries
-soroban.host-fn-op.write-ledger-byte         | meter     | number of data + code bytes in written entries
+soroban.host-fn-op.write-ledger-byte         | meter     | number of entry (data + code) bytes in written entries
 soroban.host-fn-op.write-data-byte           | meter     | number of data bytes in written entries
 soroban.host-fn-op.write-code-byte           | meter     | number of code bytes in written entries
 soroban.host-fn-op.emit-event                | meter     | number of events emitted
 soroban.host-fn-op.emit-event-byte           | meter     | number of event bytes emitted
 soroban.host-fn-op.cpu-insn                  | meter     | metered cpu instructions
 soroban.host-fn-op.mem-byte                  | meter     | metered memory bytes
-soroban.host-fn-op.invoke-time-nsecs         | meter     | time [nsecs] spent in `invoke_host_function`
-soroban.host-fn-op.cpu-insn-excl-vm          | meter     | metered cpu instructions excluding VM instantation
-soroban.host-fn-op.invoke-time-nsecs-excl-vm | meter     | time [nsecs] spent in `invoke_host_function` excluding VM instantation
+soroban.host-fn-op.invoke-time-nsecs         | timer     | time spent in `invoke_host_function`
+soroban.host-fn-op.cpu-insn-excl-vm          | meter     | metered cpu instructions excluding VM instantiation
```

Comment:
should specify -- is this per `invoke_host_function` op?

Reply:
as in, if a single `invoke_host_function`  op calls multiple VMs, is it recorded as one or multiple events?

Reply:
(we probably want 1 event per op)

Reply:
I've cleaned up this section. Yes it is per-op.